services/sidecar/src/simcore_service_sidecar/log_parser.py

```diff
@@ -101,6 +101,7 @@ async def _monitor_docker_container(
                 log_type, parsed_line = await parse_line(line)
                 await log_cb(log_type, parsed_line)
                 await writer(f"{log_type.name}: {parsed_line}")
+                await asyncio.sleep(1)
```

Comment:
?

Reply:
hmm yes. I need to double-check if that is actually needed. when I discovered the sidecar was actually never shutting down correctly I added it.

Reply:
removed!